CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 3.10)
+project(XPlanePlugin)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# X-Plane SDK path (you'll need to adjust this)
```

Comment:
The relative path for the SDK is now standardized by the team, we should have no need to adjust this